map/framework.cpp

```diff
@@ -462,6 +465,17 @@ Framework::Framework()
   LOG(LINFO, ("Editor initialized"));
 
   m_trafficManager.SetCurrentDataVersion(m_storage.GetCurrentDataVersion());
+
+#ifdef OMIM_OS_ANDROID
+  ZipFileReader::UnzipFile(GetPlatform().ResourcesDir(),
```

Comment:
Do you plan to handle exceptions here? Are you sure that it's a good idea to unzip something on the UI thread?